docs/docs/hydroflow_plus/quickstart/index.mdx

```diff
@@ -1,6 +1,42 @@
 # Quickstart
 In this tutorial, we'll walk through the basics of Hydroflow+ by building a simple dataflow that prints out the first 10 natural numbers. We'll start with a single process, then pipeline the computation, and finally distribute it across a cluster.
 
+## Installing Rust
+
+First you will need to install Rust. We recommend the conventional installation
+method, `rustup`, which allows you to easily manage and update Rust versions.
+
+[**Install Rust**](https://www.rust-lang.org/tools/install)
+
+The link in the previous line will take you to the Rust website that shows you how to
+install `rustup` and the Rust package manager `cargo` (and the
+internally-used `rustc` compiler). `cargo` is Rust's main development tool,
+used for building, running, and testing Rust code.
+
+The following `cargo` commands will come in handy:
+* `cargo check --all-targets` - Checks the workspace for any compile-time
+  errors.
+* `cargo build --all-targets` - Builds all projects/tests/benchmarks/examples
+  in the workspace.
+* `cargo clean` - Cleans the build cache, sometimes needed if the build is
+  acting up.
+* `cargo test` - Runs tests in the workspace.
+* `cargo run -p hydroflow --example <example name>` - Run an example program in
+  `hydroflow/examples`.
+
+To learn Rust see the official [Learn Rust page](https://www.rust-lang.org/learn).
+Here are some good resources:
+* [_The Rust Programming Language_, AKA "The Book"](https://doc.rust-lang.org/book/)
+* [Learn Rust With Entirely Too Many Linked Lists](https://rust-unofficial.github.io/too-many-lists/)
```

Comment:
I think this is a bit of a distraction especially for Hydroflow+, which involves pretty few interactions with borrow checking.